### PR TITLE
kmp, release-0.42: Add presubmits

### PR DESCRIPTION
**Pull request by RamLavi**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        grace_period: 5m
+      cluster: kubevirt-prow-control-plane
+      max_concurrency: 6
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20230801-94954c0
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml:22]: do you use latest bootstrap?

once the rehearse finish i will review, thanks

**Comment by oshoval** [comment]
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/3044/pull-project-infra-prow-deploy-test/1719635683292745728

/retest 

@brianmcarey 
Hi fyi
thanks

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml (hunk 1 above)
Comment (RamLavi) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml:22]: I copied from you on https://github.com/kubevirt/project-infra/pull/3042/files#diff-d22ac790b5ca665346d1a474662f8afc4fa327448518d61bf259b6014f4967c8R22

Comment (oshoval) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml:22]: seems fine
https://github.com/kubevirt/project-infra/pull/3042#issuecomment-1787236951

**Review by oshoval — APPROVED** [review]

**Review by brianmcarey — COMMENTED** [review]
/approve

**Pull request merged**